CHANGELOG.md

```diff
@@ -20,9 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Moved tests to a separate disk (tests.d64).
  - gfx: Renamed ERASE to PEN.
  - #> (the pictured numeric output string buffer) now uses its own buffer chopped from the end of PAD.
- - CHAR, HEADER and other parsing words no longer do REFILL on missing text.
  - EVALUATE now accepts multi-line strings.
  - ( refills to accept multi-line comments when parsing EVALUATE strings.
+ - CHAR, HEADER and other parsing words no longer do REFILL on missing text.
+ - Defining words based on HEADER instead abort with an error.
```

Comment:
These two lines should probably be joined.

Reply:
How about putting "CHAR doesn't REFILL" back where it was and just having the note about "HEADER etc ABORTs" be separate?

Reply:
I do not like my original wording.
How about...

 * CHAR will no longer REFILL when reaching the end of input buffer.
 * Word-creating words will now abort with an error when an unexpected end of input buffer results in an empty word name.